docs/plugins/hubspot.rst

```diff
@@ -94,11 +94,6 @@ Note, despite ``--fetch-all`` flag, the HubSpot API endpoints used in Mautic pri
 * ``/companies/v2/companies/recent/modified/``
 
 * ``/contactslistseg/v1/lists/recently_updated/contacts/recent``
-  
-If you intend to do a full sync of your HubSpot Contacts, you need to modify an attribute of each so that they appear in HubSpot's recent/modified endpoints. 
-When connecting to a long-lived HubSpot instance, these endpoints pull only Contacts modified in the last 30 days, resulting in an incomplete sync. :xref:`Source`
-
-Credit
-******
 
-:xref:`@gpassarelli` had developed this Plugin.
\ No newline at end of file
+If you intend to do a full sync of your HubSpot Contacts, you need to modify an attribute of each so that they appear in HubSpot's recent/modified endpoints.
+When connecting to a long-lived HubSpot instance, these endpoints pull only Contacts modified in the last 30 days, resulting in an incomplete sync. :xref:`Source`.
```

Comment:
I think we're missing the link file for this xref? Could you commit it @npracht? Thanks!

Reply:
Not fully clear for me how this works (much more complicated than simple .md), but it seems the link is already declared here:
![Capture d’écran 2024-12-12 à 11 27 54](https://github.com/user-attachments/assets/4d264a4c-0694-4992-a87f-b790c729db50)

Reply:
Ah sorry didn't realise it was an existing link file from the naming - thought it was a new one! All good then!